doc/install/mtls-configuration.md

```diff
@@ -17,132 +17,36 @@ At the time of writing there is [an RFC](https://github.com/Kuadrant/architectur
 
 ## Enabling mTLS
 
-### Kuadrant components
+In order to ensure that communications between the gateway and the kuadrant components are secured,
+set kuadrant's custom resource `spec.mtls.enable` field to `true`.
 
-As the Kuadrant components (Authorino & Limitador) are already part of the service mesh in Istio, mTLS can be enabled after an Envoy proxy sidecar is deployed alongside them.
```

Comment:
It's a shame to drop all this documentation. But from a user perspective it is implementation details and should not be included in a user guide. 

Thinking on some architecture doc to record implementation details like this. However, also hesitating to do so because implementation changes imply maintenance of such architecture docs